<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-J9VHH93" failures="2" tests="7" name="Test" time="60.343" errors="0" timestamp="2025-06-24T13:07:45 IST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="3.769"/>
  <testcase classname="testCases.TC002_LoginTest" name="verify_Login" time="2.184"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="7.475"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="2.549">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="4.382">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="16.703"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="7.842"/>
</testsuite> <!-- Test -->
